trakt/errors.py

```diff
@@ -32,6 +38,12 @@ def __str__(self):
         return self.message
 
 
+class BadResponseException(TraktException):
+    """TraktException type to be raised when json could not be decoded"""
+    http_code = -1
```

Comment:
used negative code for internal errors sp the positive numbers could be mapped to HTTP status codes